src/test/Simulation.t.sol

```diff
@@ -198,6 +291,59 @@ contract SimulationTests is DSTest {
         lpDaiEthPip = LPOsmAbstract(lpDaiEthPipAddr);
         linkPip = OsmAbstract(linkPipAddr);
         ethPip = OsmAbstract(ethPipAddr);
+        steCRV = GemAbstract(steCRVAddr);
+        spotter = SpotAbstract(spotterAddr);
+    }
+
+    function deployContracts() private {
+        cropManagerImp = new CropperImp(vatAddr);
+        cropManagerImpAddr = address(cropManagerImp);
+        cropManager = new Cropper();
+        cropManagerAddr = address(cropManager);
+        cropManager.setImplementation(cropManagerImpAddr);
+        CropperAbstract(cropManagerAddr).getOrCreateProxy(edAddr);
```

Comment:
This is problematic since cropManager is a contract that is sometimes treated as a proxy and sometimes as an implementation. Let's leave it the way it is. I reverted the change to apply all the suggestions in this file.